server/graphql/schema.gql

```diff
@@ -33,6 +33,15 @@ type Proposal {
   snapshot: String
 }
 
+type Vote {
+  id: String
+  timestamp: Int
+  spaceId: String
+  proposalId: String
+  choice: Int
+  voteAddress: String
```

Comment:
This should be `voter`. And voter should give you profile same as "author" in timeline like this ```
{
  voter {
    address
    profile {
      ens
      name
    }
  } 
}```